readme.md

```
@@ -1,6 +1,9 @@
-# Universal Apps | ci.dev.br
+<img src="https://apps.ci.dev.br/logo-colorida.svg">
+
+# Plataforma Universal para Aplicativos | ci.dev.br
 [Colaboradores](./docs/colaboradores.md) | [Documentação](./docs/colaboradores.md) | [Sobre]()
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)